app/scripts/svgloader.js

```diff
@@ -125,9 +125,9 @@ export const SvgLoader = {
         return Object.assign(layerData, {
           id: makeFinalNodeId_('path'),
           pathData: path,
-          fillColor: ColorUtil.svgToAndroidColor(context.fillColor, context.fillOpacity),
```

Comment:
hm, is fillOpacity never actually set? there's definitely a `fill-opacity` attribute in SVG AFAIK. We may need to multiple if it can appear in both the color and as a fill-opacity.